<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>endGame</class>
 <widget class="QMainWindow" name="endGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>319</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Questions</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>files/logo.png</normaloff>files/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(159, 62, 213);
color: rgb(255, 64, 64);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Congratulations New Highscore!!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(163, 239, 9);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitGameButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 39, 39);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
